tests/rest/admin/test_user.py

```diff
@@ -371,7 +371,8 @@ class UserRestTestCase(unittest.HomeserverTestCase):
     def prepare(self, reactor, clock, hs):
         self.store = hs.get_datastore()
 
-        self.url = "/_synapse/admin/v2/users/@bob:test"
+        self.url_user_1 = "/_synapse/admin/v2/users/@bob:test"
+        self.url_user_2 = "/_synapse/admin/v2/users/@bob2:test"
```

Comment:
given that this is only used in one test case, I would define it there rather than here